src/rest_api/api/compound_processing/compound_processing_management.rs

```diff
@@ -2,14 +2,42 @@ use poem_openapi::{payload::Json, ApiResponse, Object};
 
 #[derive(ApiResponse, Debug)]
 pub enum StandardizeResponse {
-    #[oai(status = "200")]
+    #[oai(status = "200", content_type = "application/json")]
     Ok(Json<Vec<StandardizedSmiles>>),
 }
 
+#[derive(ApiResponse, Debug)]
+pub enum ConvertedSmilesResponse {
+    #[oai(status = "200", content_type = "application/json")]
+    Ok(Json<Vec<ConvertedSmiles>>),
+}
+
+#[derive(ApiResponse, Debug)]
+pub enum ConvertedMolBlockResponse {
+    #[oai(status = "200", content_type = "application/json")]
+    Ok(Json<Vec<ConvertedMolBlock>>),
+}
+
```

Comment:
I see you moved the response wrappers to a single spot. Maybe a more neutral location would be good for all the response types?

`src/rest_api/api/response_types.rs` or similar?

Reply:
Great idea